contracts/contracts/rem.system/src/rem.system.cpp

```diff
@@ -19,7 +19,7 @@ namespace eosiosystem {
    const int64_t  default_annual_rate           = 0;       // 0% annual rate
    const int64_t  default_inflation_pay_factor  = 5;       // 20% of the inflation
    const int64_t  default_votepay_factor        = 4;       // 25% of the producer pay
-   const microseconds system_contract::vote_mature_period = eosio::days( 180 );
+   const microseconds system_contract::stake_lock_period = eosio::days( 180 );
```

Comment:
@AndriiAndrusyshyn please add new _modifiable_ constant `stake_unlock_period` with the same default value